modules/passkey/contracts/SafeWebAuthnSignerProxy.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.0;
+import {P256} from "./libraries/WebAuthn.sol";
+
+/**
+ * @title WebAuthn Safe Signature Validator
+ * @dev A proxy contracy that points to Safe signature validator implementation for a WebAuthn P-256 credential.
+ * @custom:security-contact bounty@safe.global
+ */
+contract SafeWebAuthnSignerProxy {
+    /**
+     * @notice The X coordinate of the P-256 public key of the WebAuthn credential.
+     */
+    uint256 public immutable X;
+    /**
+     * @notice The Y coordinate of the P-256 public key of the WebAuthn credential.
+     */
+    uint256 public immutable Y;
+    /**
+     * @notice The P-256 verifiers used for ECDSA signature validation.
+     */
+    P256.Verifiers public immutable VERIFIERS;
+    address internal immutable SINGLETON;
+    constructor(address implementation, uint256 x, uint256 y, P256.Verifiers verifiers) {
+        SINGLETON = implementation;
+        X = x;
+        Y = y;
+        VERIFIERS = verifiers;
+    }
+
+    /// @dev Fallback function forwards all transactions and returns all received return data.
+    // solhint-disable-next-line no-complex-fallback
```

Comment:
nit: We use `/** */` style NatSpec in the project, I would prefer for it to remain consistent.

Reply:
https://github.com/safe-global/safe-modules/pull/370/commits/954078b6179e8fea2c8b7da9774a779281d2428d